test/runner/tob.ts

```diff
@@ -48,7 +48,7 @@ export async function file2Tob(filename: string): Promise<Tob> {
   const cst = (tob.parserOptions.module ? parseModule : parseScript)(tob.input, tob.parserOptions, cb);
 
   // assert the cst is valid
-  visit(cst, tob.filename);
+  tob.parserOptions.allowTypes || visit(cst, tob.filename);
```

Comment:
what's the purpose with this option? 'allowTypes' enables parsing type annotations like TS and Flow

Reply:
let's focus on fixing JS-related issues, this is to skip walking type.

Reply:
agree. types is still on my TODO